src/components/InventoryV2/form/BatchDetailsForm.tsx

```diff
@@ -314,153 +312,67 @@ export default function BatchDetailsForm(props: BatchDetailsFormProps): JSX.Elem
-                      validateFields={validateFields}
-                    />
-                  </Grid>
-                )}
-
-                <Grid item xs={12} padding={dropdownPadding}>
-                  <AccessionsDropdown<FormRecord>
-                    availableAccessions={availableAccessions}
-                    record={record}
-                    setRecord={setRecord}
-                  />
-                  {selectedAccession && (
-                    <Box display='flex' justifyContent='space-between' flexDirection={isMobile ? 'column' : 'row'}>
-                      <Typography fontSize='14px' fontWeight={400} marginTop={1}>
-                        {strings.STATE}: {stateName(selectedAccession.state)}
-                      </Typography>
-                      {accessionQuantity === undefined && (
-                        <Typography fontSize='14px' fontWeight={400} marginTop={1}>
-                          {strings.NO_QUANTITY_SET}
-                        </Typography>
-                      )}
-                      {accessionQuantity !== undefined && (
-                        <Typography fontSize='14px' fontWeight={400} marginTop={1}>
-                          {strings.REMAINING_SEEDS}: {accessionQuantity.display}
-                        </Typography>
-                      )}
-                    </Box>
-                  )}
-                </Grid>
-
-                {['InventoryAdd', 'Species'].includes(origin) && (
-                  <Grid item xs={12} padding={dropdownPadding}>
-                    <NurseryDropdownV2
-                      availableNurseries={availableNurseries}
-                      record={record}
-                      setRecord={setRecord}
-                      validateFields={validateFields}
-                    />
-                  </Grid>
-                )}
-
-                {availableSubLocations && availableSubLocations.length > 0 && (
-                  <Grid item xs={12} padding={dropdownPadding}>
-                    <SubLocationsDropdown<FormRecord>
-                      availableSubLocations={availableSubLocations}
-                      record={record}
-                      setRecord={setRecord}
-                    />
-                  </Grid>
-                )}
-
-                {featureFlagProjects && (
-                  <Grid item xs={12} padding={dropdownPadding}>
-                    <ProjectsDropdown<FormRecord>
-                      availableProjects={availableProjects}
-                      record={record}
-                      setRecord={setRecord}
-                    />
-                  </Grid>
-                )}
-              </>
+            {featureFlagProjects && (
+              <Grid item xs={12} padding={dropdownPadding}>
+                <ProjectsDropdown<FormRecord>
+                  availableProjects={availableProjects}
+                  record={record}
+                  setRecord={setRecord}
+                />
+              </Grid>
```

Comment:
removed the conditional rendering, changes here are just indentation and removed code